changelog.md

```diff
@@ -3,6 +3,7 @@ For the latest version of this document, please see [https://github.com/bos/aeso
 #### 1.4.7.1
 
 * GHC 8.10 compatibility, thanks to Ryan Scott.
+* Fix bug in `rejectUnknownFields` not respecting `fieldLabelModifier`.
```

Comment:
I've noted that the changelog entries typically say `thanks to $Author` but I think I should not thank myself, especially while my code is under review. I hope the reviewer edits it in, if appropriate ;)

Reply:
Yeah i always go through this before release, but 1.4.7.1 is already out. We usually put new stuff under an Upcoming header, but I can do that when merging.